cmd/kubeapps-apis/proto/kubeappsapis/core/packages/v1alpha1/packages.proto

```diff
@@ -438,6 +463,61 @@ message InstalledPackageSummary {
   InstalledPackageStatus status = 11;
 }
 
+// InstalledPackageDetail
+//
+// An InstalledPackageDetail includes details about the installed package that are
+// typically useful when presenting a single installed package.
+message InstalledPackageDetail {
+  // InstalledPackageReference
+  //
+  // A reference uniquely identifying the package.
+  InstalledPackageReference installed_package_ref = 1;
+
+  // PkgVersionReference
+  //
+  // The package version reference defines a version or constraint limiting
+  // matching package versions.
+  VersionReference pkg_version_reference = 2;
+
+  // Installed package name
+  //
+  // The name given to the installed package
+  string name = 3;
+
+  // CurrentPkgVersion
+  //
+  // The version of the package which is currently installed.
+  string current_pkg_version = 4;
+
+  // ValuesApplied
+  //
+  // The values applied currently for the installed application.
```

Comment:
did you mean "installed application" or should it be "installed package" as in other places?